iteration.md

```diff
@@ -115,18 +117,18 @@ pkg: github.com/quii/learn-go-with-tests/for/v4
 PASS
 ```
 
-What `136 ns/op` means is our function takes on average 136 nanoseconds to run \(on my computer\). Which is pretty ok! To test this it ran it 10000000 times.
+`136 ns/op` significa que nossa função demora cerca de 136 nano segundos para ser executada \(no meu computador). E isso é ótimo! para chegar a esse resultado ela foi executada 10000000 vezes.
 
-_NOTE_ by default Benchmarks are run sequentially.
+_NOTE_ por padrão o benchmark é executado sequencialmente.
```

Comment:
Se "NOTE" for usada como verbo, a continuação da frase deve ser "*que* por padrão". Se for note de "NOTA", falta uma vírgula entre "padrão" e "o".